### PR TITLE
Fix build for M1 Macs (include arm64 simulator slice)

### PR DESCRIPTION
**Pull request by mluisbrown**
* replace amr64 directory name with arm64
* remove `ARCHS` option from simulator build so that arm64 slice is also built.

### CODE REVIEW

**Comment by DarthMike** [comment]
cc @MaximAlien

**Comment by mluisbrown** [comment]
Once this PR is merged, a new build of the XCFramework will need to be copied to the `Dependencies` folder of the https://github.com/zksync-sdk/zksync-swift repo.

File: build.sh [hunk 1]
```diff
@@ -11,15 +11,15 @@ file ZKSyncCrypto/libzks/libzkscrypto.a
...
             -configuration Release \
             -sdk iphonesimulator \
             -project ZKSyncCrypto.xcodeproj \
             -scheme ZKSyncCrypto \
             -archivePath ZKSyncCrypto.xcarchive \
             SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES
 
-rm -rf x86_64 
+rm -rf x86_64
 mkdir x86_64
```
Comment (MaximAlien) [inline build.sh:23]: Can you please also update this folder name as it's no longer accurate?

File: build.sh [hunk 2]
```diff
@@ -36,17 +36,17 @@ xcodebuild clean archive \
             -archivePath ZKSyncCrypto.xcarchive \
             SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES
 
-rm -rf amr64 
-mkdir amr64
+rm -rf arm64
```
Comment (MaximAlien) [inline build.sh:39]: 👍

**Review by MaximAlien — APPROVED** [review]

File: build.sh (hunk 1 above)
Comment (mluisbrown) [reply build.sh]: Fixed.

**Comment by mluisbrown** [comment]
@MaximAlien made the change you requested 👍 
I don't have rights to though 😄

**Pull request merged**